### PR TITLE
I18n sync Down & Out 06/11

### PR DESCRIPTION
**Pull request by deploy-code-org**

### CODE REVIEW

File: dashboard/config/locales/long_instructions.pt-BR.yml [hunk 1]
```diff
@@ -9655,12 +10278,93 @@ pt-BR:
...
+        adesivo bem em cima do primeiro para criar uma imagem como as mostradas abaixo.\n\n*
+        * *\n\nPressione \"Finalizar\" quando estiver satisfeito com o que você fez.\n\n![expandable](https://images.code.org/3aa37f679eabc7a5a337291db614dd98-image-1496711069351.03.49.png)\n![expandable](https://images.code.org/dd0e0eee69d37493c5ba584164316a9f-image-1553098351969.png)\n![expandable](https://images.code.org/880908899d8e6ceac8a8ce1bd0bf47eb-image-1553098355311.png)"
+      courseC_artist_loops4a_2019: |-
+        Edite seu código para que você pule para frente entre os desenhos dos dois adesivos. Neste desafio, certifique-se de ter "pular para a posição central" dentro do seu laço de repetição, para que você fique no centro!
+
+        * * *
+
+        Pressione "Finalizar" quando você estiver feliz com o que fez.
+
+        [expandable] [0]
```
Comment (bethanyaconnor) [inline dashboard/config/locales/long_instructions.pt-BR.yml:10342]: This looks like a translation issue. I'll fix this in Crowdin but wasn't going to fix it in this PR.

File: dashboard/config/locales/authored_hints.zh-CN.yml [hunk 2]
```diff
@@ -2,31 +2,39 @@
 zh-CN:
   data:
     authored_hints:
-      2-3 Artist 1 new:
+      2-3 Artist 1 new_authored_hint:
```
Comment (bethanyaconnor) [inline dashboard/config/locales/authored_hints.zh-CN.yml]: This is a weird change that only seems to be happening for zh-CN

**Comment by bethanyaconnor** [comment]
Oh the DSLS parsing failed again on zh-CN. I might need to update this PR.

**Comment by Hamms** [comment]
Looks like the apps stage of the sync out didn't complete correctly, either

File: dashboard/config/locales/blocks.es-MX.yml [hunk 3]
```diff
@@ -1468,9 +1468,9 @@ es-MX:
         text: "crear un nuevo sprite llamado {NAME} \n con el disfraz {COSTUME} \n
           en {LOCATION}"
       aalab_createNewSpriteAnon:
-        text: crear un nuevo sprite {ANIMATION_NAME} en {LOCATION}
+        text: crear un nuevo sprite {ANIMATION\_NAME} en {LOCATION}
       aalab_createNewSpriteAnonReturn:
-        text: aalab crear un nuevo sprite {ANIMATION_NAME} en {LOCATION}
+        text: aalab crear un nuevo sprite {ANIMATION\_NAME} en {LOCATION}
```
Comment (bethanyaconnor) [inline dashboard/config/locales/blocks.es-MX.yml]: Hm for some reason these escapes came back.

**Comment by bethanyaconnor** [comment]
I think this should include everything. PTAL!

File: dashboard/config/locales/blocks.es-MX.yml (hunk 3 above)
Comment (Hamms) [reply dashboard/config/locales/blocks.es-MX.yml]: I believe this will happen whenever we have to run the out step multiple times

**Review by Hamms — APPROVED** [review]

File: dashboard/config/locales/blocks.es-MX.yml (hunk 3 above)
Comment (bethanyaconnor) [reply dashboard/config/locales/blocks.es-MX.yml]: Yeah that was my theory too

**Pull request merged**